package.json

```diff
@@ -163,6 +163,12 @@
     "watch:compile": "tsc --project . --watch",
     "watch": "supervisor --watch build --quiet --exec npm -- run start"
   },
+  "bin": {
+    "atm-command": "./bin/command.js",
+    "atm-git-info": "./bin/git-info.js",
+    "atm-gql-gen": "./bin/gql-gen.js",
+    "atm-start": "./bin/start.js"
```

Comment:
I'm not the biggest fiend of `atm` as short form for `atomist` but also don't have a better idea.